src/Entities/Car.php

```diff
@@ -0,0 +1,34 @@
+<?php
+class Car 
+{
+    public int $id;
+    public string $model;
+    public int $make_id;
+    public string $make;
+    public int $bodytype_id;
+    public string $bodytype;
+    public ?string $image = null;
```

Comment:
If you make the other change I suggest to this file, you can remove setting null here since it'll be done by the constructor